src/features/customerView/CustomerViewContainer.tsx

```diff
@@ -144,6 +147,20 @@ const CustomerViewContainer = () => {
         />
       </Modal>
 
+      <Modal isOpen={!!openResendInvoice} toggleModal={() => setOpenResendInvoice(undefined)}>
+        <SendInvoiceFormContainer
+          orderId={openResendInvoice?.orderId ?? ''}
+          email={'aleksi@identio.fi'}
```

Comment:
Why do we have this email hardcoded here?

Reply:
Ask Aleksi 😅 Good catch!